src/gui/update.rs

```diff
@@ -145,8 +193,29 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
                         }
                     });
 
```

Comment:
We're missing the `ajour.catalog_header_state.columns.iter()` here, which is probably messing things up